fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/impl/FHIRPersistenceJDBCImpl.java

```diff
@@ -1521,119 +1522,167 @@ private TransactionSynchronizationRegistry getTrxSynchRegistry() throws FHIRPers
         List<ExtractedParameterValue> allParameters = new ArrayList<>();
 
         try {
-            map = SearchUtil.extractParameterValues(fhirResource);
+            if (fhirResource != null) {
```

Comment:
this is a case where I'd probably prefer `if (fhirResource == null) { return allParameters }` to keep the nesting down, although I think thats strictly personal preference

Reply:
Updated.